docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #85.                    chunk_start = max(0, chunk_stop - chunksize)
</pre><p>Similar to the original paper, we have fixed the <code>chunk_stop</code> . However, this approach is not optimal as each new chunk will contain the subsequences of the previous chunk. According to the MATLAB code, the <code>chunk_stop</code> should be updated too.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>